src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/ClassDataContract.cs

```diff
@@ -1320,12 +1318,19 @@ internal MethodInfo OnDeserialized
                 }
             }
 
-#if !NET_NATIVE
+
             internal override DataContractDictionary KnownDataContracts
             {
                 [SecurityCritical]
                 get
                 {
+#if NET_NATIVE
```

Comment:
@shmao -- question only -- could you explain why this part of the getter is NET_NATIVE only now?  I see there is code below to check once and import known data contracts.  And then it sets it into the same member field.  Under what circumstance would it be wrong for the code marked #if NET_NATIVE to execute unconditionally?